pkg/hubble/container/ring.go

```diff
@@ -24,6 +26,13 @@ import (
 	"github.com/cilium/cilium/pkg/lock"
 )
 
+var (
+	// ErrConcurrentWrite indicates that a value was overwritten
+	ErrConcurrentWrite = errors.New("concurrent write detected")
+	// ErrEndOfBuffer is an alias for io.EOF
+	ErrEndOfBuffer = io.EOF
```

Comment:
I don't think this alias is strictly necessary as `io.EOF` is widely accepted as meaning "end of whatever".
Besides, I'm still trying to understand what the "end" of a ring structure can be :)